tests/unit/suite/site/soapTest.php

```diff
@@ -1,85 +0,0 @@
-<?php
```

Comment:
this is my soap testing ground :)
I know it doesnt belong here so I will move it to my tools folder in local...